testkit/src/main/java/akka/kafka/testkit/internal/KafkaContainerCluster.java

```diff
@@ -214,11 +204,16 @@ private String readinessCheckScript() {
     String command = "#!/bin/bash \n";
     command += "set -e \n";
     command +=
-        "kafka-topics "
+        "[[ $(kafka-topics "
+            + connect
+            + " --describe --topic "
+            + READINESS_CHECK_TOPIC
+            + " | wc -l) > 1 ]] && "
+            + "kafka-topics "
             + connect
             + " --delete --topic "
             + READINESS_CHECK_TOPIC
-            + " || echo \"topic does not exist\" \n";
+            + " \n";
```

Comment:
Gracefully check that the topic exists before trying to delete it. Previous version polluted stderr when readiness check ran successfully.